biogeochem/EDPhysiologyMod.F90

```diff
@@ -1274,14 +1274,14 @@ subroutine SeedGermination( litt, cold_stat, drought_stat )
 
     !----------------------------------------------------------------------
 
-    ! germination_timescale is being pulled to PFT parameter; units are 1/yr
+    ! germination_rate is being pulled to PFT parameter; units are 1/yr
     ! thus the mortality rate of seed -> recruit (in units of carbon) 
-    ! is seed_decay_turnover(p)/germination_timescale(p)
+    ! is seed_decay_turnover(p)/germination_rate(p)
```

Comment:
Should this now be seed_decay_rate instead of seed_decay_turnover?

Reply:
yes, nice find @pollybuotte , will fix